packages/integration-tests/src/entities/Author.ts

```diff
@@ -12,6 +12,7 @@ import {
   PersistedAsyncProperty,
   Reference,
 } from "joist-orm";
+import { rangeValueRule } from "joist-orm/src/rules";
```

Comment:
Suggestion: Ideally this would be just `from "joist-orm"`, like we should go ahead and add these rules to the `src/index.ts` as exports.